src/nvim/version.c

```diff
@@ -169,7 +169,7 @@ static int included_patches[] = {
   //618,
   //617,
   //616,
-  //615,
+  //615 NA
```

Comment:
@oni-link I missed your comment at the top of #1761 showing the original patch, so I assumed that it was completely authored by you, so I'll fix.